style.css

```diff
@@ -64,7 +64,7 @@
   transform: translate(-50%, -50%);
   z-index: 10;
   background-color: #fff;
-  width: 320px;
+  width: 93vw;
   height: 99vh;
   border: 1px solid #dfe1e6;
   animation: 1s drop;
```

Comment:
- [ ] Kindly fix the issue with the buttons and the image because they are overflowing from the popup container 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/190209976-ae3708bd-14e3-4b32-aaf9-51328c9b0da6.png)

## Figma design
![image](https://user-images.githubusercontent.com/50721479/190210050-713cf660-8747-476e-8074-02079bcfa528.png)